docs/en/quantization/kv_int8.md

```diff
@@ -15,50 +15,26 @@ dequant: f = q * scale + zp
 
 ### **Step One**
 
-Convert the Hugging Face model format to the TurboMind inference format to create a workspace directory.
-
-```bash
-lmdeploy convert internlm-chat-7b /path/to/internlm-chat-7b
-```
-
-If you already have a workspace directory, skip this step.
-
-### **Step Two**
-
-Get the quantization parameters by these two steps:
+Get the quantization parameters and save them to the original HF model directory:
 
 ```bash
 # get minmax
+export HF_MODEL=internlm/internlm-chat-7b
+
 lmdeploy lite calibrate \
   $HF_MODEL \
-  --calib-dataset 'ptb' \            # Support c4, ptb, wikitext2, pileval
-  --calib-samples 128 \              # Number of samples in the calibration set, if the memory is not enough, it can be adjusted appropriately
-  --calib-seqlen 2048 \              # Length of a single text, if the memory is not enough, you can adjust it appropriately
-  --work-dir $WORK_DIR \             # Directory for saving quantized statistical parameters and quantized weights in Pytorch format
-
-# get quant parameters
-lmdeploy lite kv_qparams \
-  $WORK_DIR  \                             # Directory of the last output
-  workspace/triton_models/weights/ \       # Directory to save the quantization parameters
-  --num-tp 1  \                            # Number of GPUs used for Tensor parallelization, keep it consistent with deploy.py
+  --calib-dataset 'ptb' \
+  --calib-samples 128 \
```

Comment:
Why remove the comments?

Reply:
If one click copy code block (like [here](https://lmdeploy.readthedocs.io/en/latest/quantization/kv_int8.html#step-two))on the readthedocs, then you have to remove the comments to run it. Personally, it's not good for users when they just want to copy and run example code.

Reply:
@pppppM any comments?

Reply:
Suggest changing all `--quant-policy 4` to `--use-kv-int8 True` @RunningLeon @lvhan028